manual.php

```diff
@@ -317,17 +309,21 @@
       <br>
       <!--Unedited-->
 
-      Twin Sparrow Seal Four Dragon Seal
-      5 Ancient Kunai / 30 days 15 Ancient Kunai / 30 days
-      +10% regen rate All benefits of Twin Sparrow
-      Blue/Pink username color in chat +20% regen rate
-      Larger avatar (125x125 -> 175x175) +1 jutsu equip slot
-      Longer logout timer (60 -> 90 minutes) +1 weapon equip slot
-      Larger inbox (50 -> 75 messages) +1 armor equip slot
-      Longer journal (1000 -> 2000 characters) Enhanced long trainings (1.5x length, 2x gains)
-      Larger journal images (300x200 -> 500x500) Enhanced extended trainings (1.5x length, 2.25x gains)
-      Longer chat posts (350 -> 450 characters)
-      Longer PMs (1000 -> 1500 characters)
+      <h3>
+        Four Dragon Seal
+      </h3>
+      <h4>
+        15 Ancient Kunai / 30 days
+      </h4>
+      <div>All benefits of Twin Sparrow Seal</div>
+      <div>+20% regen rate</div>
+      <div>+1 jutsu equip slot</div>
+      <div>+1 weapon equip slot</div>
+      <div>+1 armor equip slot</div>
+      <div>Enhanced long trainings (1.5x length, 2x gains)</div>
+      <div>Enhanced extended trainings (1.5x length, 2.25x gains)</div>
+      <div>Faster stat transfers (+5/minute)</div>
+      <div>Cheaper stat transfers +100 stat points per AK</div>
```

Comment:
We could wrap this in a bullet-point list, that might make it easier to read 👍 
```suggestion
      <ul>
        <li>All benefits of Twin Sparrow Seal</li>
        <li>+20% regen rate</li>
        <li>+1 jutsu equip slot</li>
        <li>+1 weapon equip slot</li>
        <li>+1 armor equip slot</li>
        <li>Enhanced long trainings (1.5x length, 2x gains)</li>
        <li>Enhanced extended trainings (1.5x length, 2.25x gains)</li>
        <li>Faster stat transfers (+5/minute)</li>
        <li>Cheaper stat transfers +100 stat points per AK</li>
      </ul>
```